modules/post/linux/gather/keylogger.rb

```diff
@@ -0,0 +1,91 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+
+  def initialize(info={})
+    super( update_info( info,
+      'Name'          => 'Linux Keylogger',
+      'Description'   => %q{
+          Run a keylogger by reading the /dev/input/event* files.
+          You should run this as a job.
+      },
+      'License'       => MSF_LICENSE,
+      'Author'        => [ 'Eliott Teissonniere' ],
+      'Platform'      => [ 'linux' ],
+      'SessionTypes'  => [ 'meterpreter' ]
+    ))
+  end
+
+  def exists_exe?(exe)
```

Comment:
You could replace the `$PATH` parsing code with `get_path` from the `Msf::Post::Linux::System` mixin.

However, it looks like you're only using this method to check if an executable is in `$PATH` (specifically, `python`). Instead, you could use the `command_exists? 'python'` method, also from the `Msf::Post::Linux::System` mixin.

Reply:
Sure! Updated.